### ISSUE TITLE
Public quotes/reviews should be accessible outside of Bookwyrm

### ISSUE THREAD

**Issue by cloo** [bookwyrm-social/bookwyrm#2654]
Now it is not possible to read public quotes or reviews if you are not logged in to Bookwyrm.

I think the content should be available to public outside Bookwyrm, if users mark that as public, Because: 

1. Books should be for everyone, a common knowledge. We should also make discussion about them common knowledge.
2. I cannot show my reviews to people outside Bookwyrm through Bookwyrm, but I can share those quotes/reviews via Mastodon link. (yet it is broken atm.). So in terms of privacy there is no gain.
3. I find it uncanny to ask people to sign-up to a service to read my content, probably won't do that. But I'd feel delighted to show them the rich world and superb aesthetics of Bookwyrm and let them decide for themselves. Yet you don't advertise a service out of the blue, it is much nicer to say: here's what I wrote/picked about the book I mentioned, have a look.

As a possible solution, a book description for logged out users could simply include public reviews and quotes. 
Another option - adding unique shareable links for individual public reviews/quotes

**Comment by WesleyAC** [comment]
As far as I can tell, public reviews are already shown to non-logged-in users ([here's an example](https://bookwyrm.social/book/763104/s/pirate-enlightenment-or-the-real-libertalia))

Quotes don't seem to be shown to on book pages in any circumstances (except for one's own quotes), but are available via link publicly ([example](https://bookwyrm.social/user/wesleyac/quotation/925185#anchor-925185)) — these links should show up in RSS feeds, for instance.

Is this different from what you're seeing? Both of the links above work for me in private windows.

So I think this issue is mainly about having book pages show a full list of quotations — I personally would like that, but I'm not sure if there's history behind why this is the way it is.

**Comment by cloo** [comment]
Sorry I did not do proper research, guilty and appreciating your patience. You are right about the reviews - they are here, just comments and quotes are missing.

Just got a bit frustrated when wanted to show my impressions of one book with a bunch of quotes to a friend who does not have an account.  When I copied a link to that book, it did not display my public quotes.  (Of course I also secretly hoped to show her how nice Bookwyrm is). Eventually I just shared printscreens, because I even couldnt find a way to even get the individual links to quotes. Now I see they are possible somehow, but still can't find how to get those links. 

Anyways, in the described use case, my preferred experience would be to see all the public quotes as if you were logged in.
